### ISSUE TITLE
Support for namespace export as default

### ISSUE THREAD

**Issue by jdalton** [benjamn/reify#131]
I saw [this issue](https://github.com/leebyron/ecmascript-export-ns-from/issues/1) on the [namespace export proposal](https://github.com/leebyron/ecmascript-export-ns-from) and noticed it doesn't currently work with reify.
```js
export * as default from 'foo';
```

**Comment by benjamn** [comment]
What's the error?

**Comment by jdalton** [comment]
```
 SyntaxError: default is a reserved word
```

**Comment by jdalton** [comment]
Is this something we can get around by using the loose acorn parser (I think we'll end up using the loose one in the end).

**Comment by jdalton** [comment]
From acorn the error is:
```
SyntaxError: Unexpected token (3:9)
```

**Comment by jdalton** [comment]
Closed by #146.
